src/main/g8/src/main/resources/local2.conf

```
@@ -0,0 +1,6 @@
+include "application"
+include "local-shared"
+
+akka.remote.artery.canonical.port = 2552
+akka.management.http.port = 9102
+$name$.grpc.port = 8102
```

When adding these three ports I'd reverse the order (from more public to more internal) and even add some breathing space:

```suggestion
$name$.grpc.port = 8102

akka.management.http.port = 9102
akka.remote.artery.canonical.port = 2552
```